cmake/compile_definitions/macos.cmake

```diff
@@ -13,7 +13,8 @@ list(APPEND SUNSHINE_EXTERNAL_LIBRARIES
         ${CORE_MEDIA_LIBRARY}
         ${CORE_VIDEO_LIBRARY}
         ${VIDEO_TOOLBOX_LIBRARY}
-        ${FOUNDATION_LIBRARY})
+        ${FOUNDATION_LIBRARY}
+        ${APP_KIT_LIBRARY})
```

Comment:
Can you re-order this list to match the order from `./cmake/dependencies/macos.cmake`?